pom.xml

```diff
@@ -37,6 +37,13 @@
     </dependencyManagement>
 
     <dependencies>
+		<dependency>
```

Comment:
Woah. You are correct. I did check that yesterday but did not get a visual indication tabs where in the file. All fixed now.